sshd-core/src/main/java/org/apache/sshd/client/session/ClientProxyConnector.java

```diff
@@ -31,12 +31,13 @@
 @FunctionalInterface
 public interface ClientProxyConnector {
     /**
-     * Invoked just before the client identification is sent so that the
-     * proxy can send the meta-data to its peer. Upon successful return
-     * the SSH identification line is sent and the protocol proceeds as usual.
+     * Invoked just before the client identification is about to be sent so that
```

Comment:
This comment is not true anymore. It's called just before the SSH protocol starts.